### PR TITLE
Finish job page

### PR DESCRIPTION
**Pull request by mpolimen**
TODO's (good to have, not necessary crucial):

- Delete button for each note? Would be easy, but not done in the backend yet...
- Sorting application events for timeline (backend or frontend?)

### CODE REVIEW

File: client/src/api/routes/applications.ts [hunk 1]
```diff
@@ -137,24 +133,29 @@ export const useUpdateApplication = (applicationId: string, company?: string, po
     });
 };
 
+export const useCreateNote = (applicationId: string, note?: string): UseQueryResult<ApplicationObject> => {
```
Comment (roccopolimen) [inline client/src/api/routes/applications.ts]: createNote should not have the note as an optional parameter, for a patch yes but not post

File: client/src/api/routes/applications.ts [hunk 2]
```diff
@@ -137,24 +133,29 @@ export const useUpdateApplication = (applicationId: string, company?: string, po
...
  * @param {string} eventId 
- * @param {string} status 
- * @param {string} title 
- * @param {string} date 
- * @param {string} location 
+ * @param {string} status
  * @returns {UseQueryResult<EventObject>} the updated event
  * @throws if fails
  */
-export const useUpdateEvent = (applicationId: string, eventId: string, status?: boolean, title?: string, date?: string, location?: string): UseQueryResult<EventObject> => {
```
Comment (roccopolimen) [inline client/src/api/routes/applications.ts:151]: Don't agree with this design choice to force users to delete an event and create a new one if they want to change crucial fields such as `title`, `date`, `location`. These are fields I would be editing the most and I would hate to have to make a new one each time.

File: client/src/components/JobSections/MyNotes/index.tsx [hunk 3]
```diff
@@ -0,0 +1,62 @@
+import { Box, Button, Card, CardContent, Grid, TextField, Typography } from "@mui/material";
+import { useEffect, useState } from "react";
+import { ApplicationObject } from "typings";
+
+
+export default function MyContacts(props: {
```
Comment (roccopolimen) [inline client/src/components/JobSections/MyNotes/index.tsx]: name? Should be MyNotes I believe

File: server/routes/applications.ts [hunk 4]
```diff
@@ -80,6 +80,37 @@ router.post('/', async (req, res) => {
     }
 });
 
+// POST /note/:applicationId
+router.post('/note/:applicationId', async (req, res) => {
+    console.log(req.body); // TODO: remove
```
Comment (roccopolimen) [inline server/routes/applications.ts]: TODO: remove

**Review by roccopolimen — COMMENTED** [review]
Few comments, mostly minor except for the one design decision I disagree with

File: client/src/api/routes/applications.ts (hunk 2 above)
Comment (mpolimen) [reply client/src/api/routes/applications.ts:151]: For now, this is just to keep things simple. It makes this manageable for now. Could be a last minute quality of life addition to do what you want. I will not do it in this PR.

**Comment by GraceMattern** [comment]
Looks good overall

**Comment by roccopolimen** [comment]
Merging

**Pull request merged**